Automattic/wp-calypso
# ETK: Remove unusued block patterns modifications

#### Changes proposed in this Pull Request

* Following on from #50111 where we disabled the premium block patterns feature due to performance concerns, this PR removes the feature altogether. We haven't yet had a chance to revisit re-implementing the feature in a performant way, so for now, let's tidy up the ETK plugin by stripping out the code, which hasn't been running anyway.

#### Testing instructions



* Apply this change (e.g. from a sandbox using `install-plugin.sh etk update/remove-block-patterns-modifications-from-etk`)
* Test in the editor that nothing breaks, and that you can still see block patterns being loaded from wpcom (e.g. there are patterns in the Featured category)
* Double-check visually that I haven't missed anything / removed anything that I shouldn't

Note that there are currently some duplicate patterns loading from .org, which will be resolved by D64053-code, and is unrelated to this PR.



Related to 236-gh-Automattic/view-design
